# Representation of a (desired) pose and velocity 
# std_msgs/Header header
std_msgs/Float64 time
geometry_msgs/Twist e
geometry_msgs/Twist ed
float64 e_lin_norm
float64 e_ang_norm
float64 ed_lin_norm
float64 ed_ang_norm